server/migrations/helpers.test.js

```diff
@@ -15,59 +16,13 @@ describe(calculatePreferences, () => {
         config: 'foo',
       },
     }
-    expect(calculatePreferences(user)).toEqual({
-      config: 'foo',
-      sort: 0,
-      star: [
-        { site: 'CA', starredSubjects: [] },
-        { site: 'LA', starredSubjects: [] },
-      ],
-      complete: [{ site: 'CA', completedSubjects: [] }],
-    })
-  })
```

Comment:
We have this condition in the code now so this condition would be skipped over.
https://github.com/AMP-SCZ/dpdash/pull/241/files#:~:text=(user)-,const%20isUserMigrated%20%3D%20!!user.siteSubjects%3F.length,%7D,-%7D)

Reply:
there are no site subjects for this user though so doesn't that mean this won't be skipped over?

Reply:
These conditions are being counted here https://github.com/AMP-SCZ/dpdash/pull/241/files#:~:text=%27636d1b784723201176bf0bda%27%2C-,siteSubjects%3A%20%5B,starredSubjects%3A%20%5B%5D%2C%20completedSubjects%3A%20%5B%5D%20%7D%2C,-%7B
The second test was removed since we won't reach it because of the condition

Reply:
I'm not following. We're adding more logic in the helper function and removing test examples. I don't see how the one test example we have left gives us enough coverage for the logic in these helpers. Am I missing the coverage somewhere else?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241#:~:text=%E2%80%A6-,4ea6d92,-Merge%20state